src/components/ResourceList/ResourceList.tsx

```diff
@@ -53,6 +53,12 @@ export interface ResourceListProps {
   /** Item data; each item is passed to renderItem */
   items: Items;
   filterControl?: React.ReactNode;
+  /** The markup to display when no resources exist yet. Renders when set and items is empty. */
+  emptyState?: React.ReactNode;
+  /** The markup to display when no results are returned on search or filter of the list. Renders when `filterControl` is set, items are empty, and `emptyState` is not set.
```

Comment:
Could someone have filterControls on a list that has not had resources yet?

Reply:
Yep! You can see what that looks like in the playground code--aside from making list views more familiar to new merchants, the reason for moving to in-context empty states is for smooth transition between loading and empty or with-data states 😊.